pkg/cli/cli_test.go

```diff
@@ -402,9 +402,9 @@ func Example_ranges() {
 	// 	0: node-id=1 store-id=1
 	// /System/"tse"-"ranges3" [4]
 	// 	0: node-id=1 store-id=1
-	// "ranges3"-/Table/0 [11]
+	// "ranges3"-/Table/SystemConfigSpan/Start [11]
```

Comment:
There's a special case in the key pretty-printer for the start of the system config span (which would otherwise look like `/Table/0/0`). We try to split at that key, but before this change, EnsureSafeSplitKey changed the split key to `/Table/0`. Now the split point correctly shows that it is `/Table/0/0` or its alias `/Table/SystemConfigSpan/Start`.